bindings/pydeck/examples/grid_layer.py

```diff
@@ -0,0 +1,29 @@
+import pydeck as pdk
+import pandas as pd
+
+GPUGRID_LAYER_DATA = ('https://raw.githubusercontent.com/uber-common/'
+                      'deck.gl-data/master/website/sf-bike-parking.json')
+df = pd.read_json(GPUGRID_LAYER_DATA)
+
+# Define a layer to display on a map
+
+layer = pdk.Layer(
+    'GridLayer',
+    df,
+    pickable=True,
+    extruded=True,
+    cell_size=200,
+    elevation_scale=4,
+    get_position='COORDINATES',
```

Comment:
Should we use the function syntax here `@=` @ajduberstein  can you point to the docs?

Reply:
It's @@= is actually used under the hood, (`COORDINATES` is converted to `@@=COORDINATES` before being passed to the JSON-to-props converter) so this should be fine. Users have to be explicit when they want string constants (see #4270), not variable names.